src/components/form/examples/row-layout-schema.ts

```diff
@@ -0,0 +1,107 @@
+export const schema = {
+    description: 'This form has the row layout',
+    type: 'object',
+    properties: {
+        info: {
+            title: 'Preferences',
+            description:
+                'These settings will not affect how others see the data. These are only for you.',
+            type: 'object',
+            lime: {
+                layout: {
+                    type: 'row',
+                },
+            },
+            properties: {
+                language: {
+                    type: 'string',
+                    icon: 'globe',
+                    title: 'Language',
+                    description: 'Select the app language',
+                    default: 'ua',
+                    lime: {
+                        component: 'limel-select',
+                    },
+                    oneOf: [
```

Comment:
![Screenshot 2023-02-07 at 16 15 29](https://user-images.githubusercontent.com/35954987/217285506-863af66c-5d77-4579-b45d-aaaff60359a2.png)
It would be nice to inject this class on `limel-select` if it is used in a form with `row` layout. Is it possible?
Because otherwise they render mega big!

Or is it the responsibility of the consumer to add is a class via schema or via a custom component? @jgroth

Reply:
It seems weird that the select component renders that big anyway… is there any case where we actually want that behaviour? Or is it a side-effect of some other behaviour we _do_ want? If not, maybe we can just make it not do that in the first place, and not need the extra class? 🤔

Reply:
I'm not sure there is an easy way to add the class the way things work now :thinking: One way would be to inspect the schema and inject the class that way, but it's not the prettiest solution. Another way could be to set the class in the react wrapper class for the select component, not sure if that is pretty or easy to do either.

I guess this would have been a lot easier to solve if the narrow style was applied using css properties instead, but not sure if that is possible to do or not :slightly_smiling_face:

Reply:
> It seems weird that the select component renders that big anyway…

All form fields (input, slider, select, etc…) have the same height, otherwise they'd look horrible on the Overview tab or other forms with a grid layout. This row layout is kind of an exception now.

> I'm not sure there is an easy way to add the class the way things work now 

Then I think we leave it for the consumers. They can always do so. 

> if the narrow style was applied using css properties instead

That's unfortunately not so easy right now. Many things change in the `is-narrow` UI. That might require a lot of refactoring. However, I have been thinking about enabling different densities in Lime Elements, using CSS props. Not sure how yet. But maybe some beautiful day we can find a solution.